src/runtime_src/core/tools/xbmgmt2/flash/xspi.h

```diff
@@ -42,8 +42,8 @@ class XSPI_Flasher
  public:
```

Comment:
Copyright needs to be updated. A new line after 3 should be added to reflect AMD copyright. For example:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`